rocketpy/rocket/aero_surface.py

```diff
@@ -1889,3 +1892,210 @@ def all_info(self):
         """
         self.prints.all()
         return None
+
+
+class AirBrakes(AeroSurface):
+    """AirBrakes class. Inherits from AeroSurface.
+
+    Attributes
+    ----------
+    AirBrakes.drag_coefficient : Function
+        Drag coefficient as a function of deployment level and Mach number.
+    AirBrakes.drag_coefficient_curve : int, float, callable, array, string, Function
+        Curve that defines the drag coefficient as a function of deployment level
+        and Mach number.  Used as the source of `AirBrakes.drag_coefficient`.
+    AirBrakes.deployment_level : float
+        Current deployment level, ranging from 0 to 1. Deployment level is the
+        fraction of the total airbrake area that is deployed.
+    AirBrakes.reference_area : int, float
+        Reference area used to calculate the drag force of the air brakes
+        from the drag coefficient curve. Units of m^2.
+    AirBrakes.clamp : bool, optional
+        If True, the simulation will clamp the deployment level to 0 or 1 if
+        the deployment level is out of bounds. If False, the simulation will
+        not clamp the deployment level and will instead raise a warning if
+        the deployment level is out of bounds. Default is True.
+    AirBrakes.name : str
+        Name of the air brakes.
+    """
+
+    def __init__(
+        self,
+        drag_coefficient_curve,
+        reference_area,
+        clamp=True,
+        substitute_rocket_drag_coefficient=False,
```

Comment:
This argument name is tooooo long, it starts to get annoying somehow.

Can we change to `override_cd`?

Reply:
I'll set with `override_rocket_drag` if thats good enough for you

Reply:
Well done!